### PR TITLE
Disposition

### PR DESCRIPTION
**Pull request by riamon-v**
I'm trying to rework the team section as best as possible to make it look smaller on responsive device and less impressive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,122 +183,118 @@ <h3 class="section-title">Contact</h3>
...
-                        <div class="member">
-                            <div class="pic"><img src="img/Team/Jakob.png" alt=""></div>
-                            <h4>Jakob Kellendonk</h4>
-                            <span>Chief Technic Officer</span>
-                            <div class="social">
-                                <a target="_blank" href="https://github.com/Gaspard--"><i class="fab fa-github"></i></a>
-                                <!--<a href=""><i class="fa fa-linkedin"></i></a>-->
-                            </div>
+                    <div class="member-top">
+                      <div class="pic"><img src="img/Team/Flora.png" alt="FLora Huot, UX Designer et developer on RAVEN-OS"></div>
```
Comment (person-m) [inline index.html]: FLora -> Flora

File: index.html [hunk 2]
```diff
@@ -183,122 +183,118 @@ <h3 class="section-title">Contact</h3>
...
                                 <a target="_blank" href="https://www.linkedin.com/in/dorian-trubelle-341a91114/"><i class="fab fa-linkedin"></i></a>
                             </div>
                         </div>
-                    </div>
-                    <div class="col-lg-2 col-md-4 col-xs-6">
-                        <div class="member">
-                            <div class="pic"><img src="img/Team/Melvin.png" alt=""></div>
+
+                        <div class="member-bottom">
+                            <div class="pic"><img src="img/Team/Melvin.png" alt="Mevlin Personnier, web developer for Leaves on RAVEN-OS"></div>
```
Comment (person-m) [inline index.html]: Mevlin -> Melvin

File: index.html [hunk 3]
```diff
@@ -183,122 +183,118 @@ <h3 class="section-title">Contact</h3>
...
-                            <h4>Vincent Riamon</h4>
-                            <span>Feathers Developer</span>
-                            <div class="social">
-                                <a target="_blank" href="https://github.com/riamon-v"><i class="fab fa-github"></i></a>
-                                <a target="_blank" href="https://www.linkedin.com/in/vincent-riamon-27626a127/"><i class="fab fa-linkedin"></i></a>
-                            </div>
+                    <div class="member-top">
+                        <div class="pic"><img class="img-responsive" src="img/Team/Vincent.png" alt="Vincent Riamon, Developer Feathers and Leaves on RAVEN-OS"></div>
+                        <h4>Vincent Riamon</h4>
+                        <span class="too-long">Leaves Developer</span>
```
Comment (person-m) [inline index.html]: The class is "too-long" for Vincent but it's not for Melvin and Mickaël

File: index.html [hunk 4]
```diff
@@ -183,122 +183,124 @@ <h3 class="section-title">Contact</h3>
...
-                            <h4>Benjamin Grange</h4>
-                            <span>Chief Executive Officer</span>
-                            <div class="social">
-                                <a target="_blank" href="https://github.com/Arignir"><i class="fab fa-github"></i></a>
-                            </div>
+                <div class="row row-team">
+                  <div class="col-sm-11 content-row-team">
+                    <div class="member-top">
+                      <div id="Benjamin">
+                        <div class="pic"><img src="img/Team/Benjamin.png" alt="Benjamin Grange, Chief Executive Officer, developer Nest on RAVEN-OS"></div>
```
Comment (Arignir) [inline index.html]: It's should be written `Raven-OS` :p

**Pull request merged**